src/wasm.h

```diff
@@ -821,6 +821,7 @@ class CallIndirect : public SpecificExpression<Expression::CallIndirectId> {
   Signature sig;
   ExpressionList operands;
   Expression* target;
+  Name tableName;
```

Comment:
```suggestion
  Name table;
```
I think that's more consistent with other terminology.

Reply:
Thanks. Done.